components/HelpButton.tsx

```diff
@@ -129,12 +129,14 @@ const helpText = (screenName: ScreenName) => {
     case "DownloadMapScreen":
       return "Når du er ute på oppsynstur kan det hende at du ikke har mobildekning. Kartet må derfor lastes ned på forhånd.\n\n\
```

Comment:
Siden de nå kan skippe nedlasting
```suggestion
      return "Når du er ute på oppsynstur kan det hende at du ikke har mobildekning. Kartet kan derfor lastes ned på forhånd.\n\n\
```